tests/validators/test_complex.py

```diff
@@ -84,7 +85,7 @@ def test_complex_strict(input_value, expected):
 
 
 @pytest.mark.xfail(
-    platform.python_implementation() == 'PyPy',
+    platform.python_implementation() == 'PyPy' and sys.version_info < (3, 10),
```

Comment:
For the record, I think testing `sys.pypy_version_info < (7, 3, 17)` would be more correct.

Reply:
@mgorny, good point, could you please open a PR?

Reply:
I've quickly filed #1441 but I have to run now (before CI is done).